web/src/components/task/task-sidebar/finish-button/utils.ts

```diff
@@ -1,7 +1,29 @@
+const LISTED_PAGES_LIMIT = 10;
+const SLICE_START_INDEX = 0;
+const SLICE_END_INDEX = LISTED_PAGES_LIMIT - 1;
```

Comment:
Are you sure we need this "-1" here? As I understand, in this case we'll show 9 numbers instead of 10 and 1 page number will be lost when we do "pages.length - LISTED_PAGES_LIMIT".

Reply:
Addressed